<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulación Gravitacional</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout" stretch="10,1">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <widget class="QPushButton" name="btnSistema1">
        <property name="text">
         <string>Sistema 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSistema2">
        <property name="text">
         <string>Sistema 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSistemaPersonalizado">
        <property name="text">
         <string>Sistema Personalizado</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnIniciar">
        <property name="text">
         <string>Iniciar Simulación</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGuardar">
        <property name="text">
         <string>Guardar Datos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReiniciar">
        <property name="text">
         <string>Reiniciar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>